### DISCUSSION TITLE
Questions about write into csv files

### DISCUSSION THREAD

**Discussion by Elaine77777** [Elaine77777/lab-05#1] [General]
Hi Dr. Francom, I'm trapped by the last step in lab-05, which is to write everything I downloaded into a csv file, but my Rstudio doesn't allow me to do it, saying that it cannot open the file for writing. Here's my code, and I wonder what does it mean by cannot open the file for writing: 
```{r}
  works_info_tbl <-
  gutenberg_download(gutenberg_id = works_info_tbl$gutenberg_id,
    meta_field = c("title", "author", "gutenberg_id"))


  write_csv(x = works_info_tbl, file = "../data/original/gutenberg/works_info_tbl.csv")
```

To figure out what is the problem, instead of using my code, I directly copied all the codes in Recipe 5, the example you gave us about downloading all the books from 1800 to 1880, but then, in the console, it says that: "Cannot open file for writing:
* 'data/original/gutenberg/works.csv'" "Warning: ! Could not download a book at http://aleph.gutenberg.org/4/46/46.zip.
ℹ The book may have been archived.
ℹ Alternatively, You may need to select a different mirror.
→ See https://www.gutenberg.org/MIRRORS.ALL for options.Error: Cannot open file for writing:
* 'data/original/gutenberg/works.csv'"
Here's the code for this one: 
```{r}
# Download works
works |>
  filter(
    rights == "Public domain in the USA.",
    has_text == TRUE
  ) |>
    slice_head(n = 5) |>
  gutenberg_download(
    meta_fields = c("title", "author", "gutenberg_author_id", "gutenberg_bookshelf")
  ) |>
 write_csv(file = "data/original/gutenberg/works.csv")
```

I wonder what is the problem here. I assume it should be something relevant to the way I downloaded Gutenberg in the first place? 

Thank you!

@francojc

**Comment by francojc** [comment]
I added a pull request to your lab-05 repo that addresses these issues. Take a look and let me know if you have any questions.
